perma_web/perma/celery_tasks.py

```diff
@@ -1242,32 +1243,64 @@ def seconds_to_minutes(seconds_val):
         return f"{math.ceil(seconds_val)} seconds"
 
 
-def save_warc_to_temp(input_guid, storage_file):
+def save_warc_for_conversion(input_guid, storage_file):
     """
     Gets the file path
     Get the associated file's contents from storage
     Saves as temp file
     """
     contents = default_storage.open(storage_file).read()
 
-    with tempfile.NamedTemporaryFile(delete=False, suffix=f"_{input_guid}.warc.gz") as temp_warc_file:
-        temp_warc_file.write(contents)
+    with open(f"{input_guid}.warc.gz", "wb") as file:
+        file.write(contents)
 
-    return temp_warc_file.name
+    return file.name
 
 
-def create_wacz_temp_path(input_guid):
+def create_wacz_path(input_guid):
     """
     Creates temp file for wacz
     """
-    wacz_suffix = f"_{input_guid}.wacz"
-    with tempfile.NamedTemporaryFile(mode='w', delete=False, suffix=wacz_suffix) as temp_wacz_file:
+    with open(f"{input_guid}.wacz", "wb") as file:
         pass
 
-    return temp_wacz_file.name
+    return file.name
+
+
+def create_pages_jsonl(link_object):
+    """
+    Creates pages.jsonl file to pass to js-wacz call
+    """
+    jsonl_rows = []
+
+    attributes_to_check = [
+        {"capture_type": link_object.primary_capture, "title": "primary_capture", "creation_ts":
```

Comment:
Nice to have: Since pages.jsonl is mostly a user interface, it could be nice to give more human-readable titles to these entry points. I.E: "Screenshot for {url}".